docs/source/tools/assistant.rst

```diff
@@ -184,3 +184,12 @@ Rarely, you might need to install an SDK release from a downloaded SDK release t
 
   daml install path-to-tarball.tar.gz
 
+Terminal Command Completion
+***************************
+
+The ``daml`` assistant comes with support for ``bash`` completions. These will be installed automatically on Linux and Mac when you install or upgrade the DAML assistant. If you use the ``bash`` shell, and your ``bash`` supports completions, you can use the TAB key to complete many ``daml`` commands, such as ``daml install`` and ``daml version``.
+
+You can override whether bash completions are installed for ``daml`` by
+passing ``--bash-completions=yes`` or ``--bash-completions=no`` to ``daml install``.
+
+In the future, ``zsh`` and ``fish`` terminal completions may be implemented.
```

Comment:
Please don't promise or even hint at future functionality in the docs.

Reply:
While I agree with you, I’ve already started to look into ZSH completions and have them working locally and we know what we need to do for fish so this might even land before the next release.

Reply:
Removed.